libraries/namespaces.raml

```diff
@@ -0,0 +1,7 @@
+#%RAML 1.0 Library
+annotationTypes:
+  namespace:
+    description: |
+      this annotation type allows you to associate unique identifier with a RAML document.
+    type: string
+(namespace): http://raml.org/vocabularies/annotations/core
```

Comment:
>That would potentially mean that specs that using this annotation might fail
Not actually, because specification include some fixed version of the library.
>Why are we not making it restrictive now and if use cases come in that need a bit more flexibility, we open it?
Already agreed and changed to `uri`. :-)